client/src/main/java/io/pravega/client/netty/impl/FlowHandler.java

```diff
@@ -237,6 +269,10 @@ public void close() {
                 if (openFlowCount != 0) {
                     log.warn("{} flows are not closed", openFlowCount);
                 }
+                final int appendTrackerCount = flowIDBatchSizeTrackerMap.size();
+                if (appendTrackerCount != 0) {
+                    log.warn("{} AppendBatchSizeTrackers are not closed", appendTrackerCount);
```

Comment:
So why are you not closing them?

Reply:
ok.. i will check this. I guess., connection is getting closed only when flow id count reaches 0. however i check , if required i post another JIIRA